client/src/components/UserProfile/UserProfileContainer.js

```diff
@@ -2,14 +2,43 @@ import React from 'react'
 import UserPreferencesContainer from './UserPreferencesContainer'
 import UserBioContainer from './UserBioContainer';
 import './UserProfile.css'
+import { withRouter } from 'react-router-dom';
+import { Query } from 'react-apollo'
 
-const UserProfileContainer = () => {
-	return (
-		<div className="user-profile-container">
-			<UserBioContainer />
-			<UserPreferencesContainer />
-		</div>
-	)
+import GET_USER_BIO from './queryGetUserBio.graphql';
+import GET_USER_PREFS from './queryGetUserPrefs.graphql';
+
+class UserProfileContainer extends React.Component {
+	render(){
+		const { id } = this.props.match.params
+		return (
+			<div className="user-profile-container">
+				<Query query={GET_USER_BIO} variables={{ id }}>
+						{({ data, loading, error }) => {
+							if (loading) return <p>Loading</p>;
+							if (error) return <p>ERROR</p>;
+							const userBio = data.getUser
+							console.log(userBio)
+							return (
+								<UserBioContainer userBio={userBio}/>
+						)
+					}}
+				</Query>
+				<Query query={GET_USER_PREFS} variables={{ id }}>
```

Comment:
why not use just one query?

Reply:
I was trying to use the same component logic while making my queries so each query returns a single request. Let me know if it's not a suitable approach for this, can change it